packages/mcp-docs-server/package.json

```diff
@@ -0,0 +1,100 @@
+{
+  "name": "@mastra/mcp-docs-server",
+  "version": "0.0.1-alpha.0",
+  "description": "MCP server for accessing Mastra.ai documentation, changelogs, and news.",
+  "type": "module",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "bin": "dist/stdio.js",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      },
+      "require": {
+        "types": "./dist/index.d.cts",
+        "default": "./dist/index.cjs"
+      }
+    },
+    "./package.json": "./package.json"
+  },
+  "scripts": {
+    "prepare-docs": "tsx src/prepare-docs/index.ts",
+    "prebuild": "pnpm prepare-docs",
+    "build": "tsc",
+    "predev": "pnpm prepare-docs",
+    "dev": "fastmcp dev src/index.ts",
+    "inspect": "fastmcp inspect src/index.ts",
+    "build:watch": "pnpm build --watch",
+    "pretest": "pnpm build",
+    "test": "vitest run",
+    "lint": "eslint ."
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@mastra/core": "workspace:^",
+    "@modelcontextprotocol/sdk": "^1.7.0",
+    "date-fns": "^4.1.0",
+    "exit-hook": "^4.0.0",
+    "fastmcp": "^1.20.4",
+    "jsdom": "^26.0.0",
+    "turndown": "^7.1.2",
+    "tylerbarnes-fastmcp-fix": "^1.0.0",
+    "uuid": "^11.1.0",
+    "zod": "^3.22.4"
+  },
+  "devDependencies": {
+    "@ai-sdk/anthropic": "^1.1.15",
+    "@internal/lint": "workspace:*",
+    "@mastra/astra": "workspace:^",
+    "@mastra/chroma": "workspace:^",
+    "@mastra/client-js": "workspace:^",
+    "@mastra/composio": "workspace:^",
+    "@mastra/core": "workspace:^",
+    "@mastra/deployer": "workspace:^",
+    "@mastra/deployer-cloudflare": "workspace:^",
+    "@mastra/deployer-netlify": "workspace:^",
+    "@mastra/deployer-vercel": "workspace:^",
+    "@mastra/evals": "workspace:^",
+    "@mastra/firecrawl": "workspace:^",
+    "@mastra/github": "workspace:^",
+    "@mastra/loggers": "workspace:^",
+    "@mastra/mcp": "workspace:^",
+    "@mastra/memory": "workspace:^",
+    "@mastra/pg": "workspace:^",
+    "@mastra/pinecone": "workspace:^",
+    "@mastra/playground-ui": "workspace:^",
+    "@mastra/qdrant": "workspace:^",
+    "@mastra/rag": "workspace:^",
+    "@mastra/ragie": "workspace:^",
+    "@mastra/speech-azure": "workspace:^",
+    "@mastra/speech-deepgram": "workspace:^",
+    "@mastra/speech-elevenlabs": "workspace:^",
+    "@mastra/speech-google": "workspace:^",
+    "@mastra/speech-ibm": "workspace:^",
+    "@mastra/speech-murf": "workspace:^",
+    "@mastra/speech-openai": "workspace:^",
+    "@mastra/speech-playai": "workspace:^",
+    "@mastra/speech-replicate": "workspace:^",
+    "@mastra/speech-speechify": "workspace:^",
+    "@mastra/stabilityai": "workspace:^",
+    "@mastra/turbopuffer": "workspace:^",
+    "@mastra/upstash": "workspace:^",
+    "@mastra/vectorize": "workspace:^",
```

Comment:
@wardpeet any ideas here? I did this so it will always bump/release with any other package - but downside of course is the users machine will install all these packages. I couldn't see a way in the changesets documentation to have a package like this where the version bumps with any other package